rdr_service/resource/generators/participant.py

```diff
@@ -569,6 +569,15 @@ def _prep_modules(self, p_id, ro_session):
             filter(QuestionnaireResponse.questionnaireId ==
                    QuestionnaireConcept.questionnaireId).label('codeId')
 
+        # TODO: Workaround for PDR-364 is to pull cohort value from participant_summary. LIMITED USE CASE ONLY
```

Comment:
Can we move this into `_prep_participant_profile()` ?  We're already querying ParticipantSummary there;  just need to add the consentCohort column to the list of returned columns?

We already have some other spots where we query ParticipantSummary but it may help later on to consolidate them as much as possible?

Reply:
Sounds good, I'll do that.